packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/document_migrator.test.ts

```diff
@@ -1329,6 +1342,175 @@ describe('DocumentMigrator', () => {
       });
     });
   });
+
+  describe('down transformation', () => {
```

Comment:
Most of the unit tests are directly for `downgrade_pipeline`. In theory, I should have stubbed the pipeline for the doc migrator tests, but
1. we're not doing it for other unit tests in the file and the 'normal' pipeline
2. this part is so touchy that including the underlying implementation in the testing makes some sense

So I added some 'integration' smoke tests here.